### PR TITLE
Corrections in the envelop

### PR DESCRIPTION
**Pull request by markella-n-th**
(Thanks for sending a pull request! Please make sure you click the link above to view the [contribution guidelines](CONTRIBUTING.md), then fill out the blanks below.)

Adding:
- Source information
- Sink information
- Consent information
Changes:
- The session token is moved one level up.
- Corrections to pass the validation tests.

**What does this implement/fix? Explain your changes.**

...

**Does this close any currently open issues?**

...

**Any relevant logs, error output, etc?**

...

**Any other comments?**

...

**Where has this been tested?**
 - **Device:** [e.g. iPhone6]
 - **OS:** [e.g. iOS8.1]
 - **Browser:** [e.g. stock browser, safari]
 - **Version:** [e.g. 22]

### CODE REVIEW

**Comment by ixuz** [comment]
Awaiting changes from https://github.com/MagnumOpuses/common-cv-model/pull/30

File: envelope/DataEnvelope.json [hunk 1]
```diff
@@ -66,7 +126,7 @@
                     "type": "int"
                 },
                 "data": {
-                    "description": "this is the data transferred.",
+                    "description": "The data transferred.",
                     "type": "structure",
                     "allOf": [
                         { "$ref": "../common data structure/recruiting/json/CandidateType.json#" }
```
[the remark this answers is not in the document]
Comment (markella-n-th) [reply envelope/DataEnvelope.json:132]: will correct as soon as possible.

[the remark this answers is not in the document]
Comment (markella-n-th) [reply envelope/DataEnvelope.json:132]: Actually this is corrected int he later code so merging the branch with the latest changes to stage should fix it.

[the remark this answers is not in the document]
Comment (ixuz) [reply envelope/DataEnvelope.json]: Relevant corrections have now been merged into stage, resolving this.

**Pull request merged**